### PR TITLE
Add usage samples

### PR DESCRIPTION
**Pull request by navalev**
1. Add postman collection with REST API usage samples
2. Samples docs minor update

### CODE REVIEW

**Review by omri374 — APPROVED** [review]

File: docs/development.md [hunk 1]
```diff
@@ -9,78 +9,77 @@
...
 The project is structured so that:
-- Each Preisio service has a designated directory. In it, the source code for:
-    - The service logic.
-    - Tests, both unit and integration.
-    - Serving it as an HTTP service (found in app.py).
-    - Python Packaging setup script (setup.py).
-- In the project root directory, you will find common code for using, serving and testing Presidio
-as a cluster of services, as well as CI/CD pipelines codebase and documentation.
+
+-   Each Preisio service has a designated directory. In it, the source code for:
```
Comment (shiranr) [inline docs/development.md:13]: presidio

Comment (shiranr) [inline docs/development.md:13]: Would replace this to:
Each Presidio service has a designated directory containing:

File: docs/samples/index.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
-# Usage samples
+# Samples
 
-## Table of contents
+## Usage Samples
+
+1. [Running in a Python pipeline](python/index.md)
```
Comment (shiranr) [inline docs/samples/index.md:5]: This links are currently not working... need to verify they are.

**Review by shiranr — COMMENTED** [review]
Looks good small comments

File: docs/samples/index.md (hunk 2 above)
Comment (navalev) [reply docs/samples/index.md:5]: I've checked again and the links worked. where did u validate this?

**Review by SharonHart — APPROVED** [review]

File: docs/samples/index.md (hunk 2 above)
Comment (shiranr) [reply docs/samples/index.md:5]: in github readme... might be that's the problem.

**Pull request merged**